ADDING_AN_EXAMPLE.md

```diff
@@ -0,0 +1,59 @@
+# How to Add a New Example
+
+Ideally, each example should come with both Rust and Motoko implementations,
+implementing the same Candid interface (and, ideally, semantics).
```

Comment:
One exception is the variant type. Rust prefers to capitalize the first character of variant tags, while Motoko prefers to use lower case.